<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>661</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mudir MT</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
border-color: rgb(239, 240, 242);
background-color: rgb(239, 240, 242);
selection-color: rgb(239, 240, 242);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(0, 0, 0);
gridline-color: rgb(6, 6, 6);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>S'authentifier</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>210</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(65, 195, 95);
background-color: rgb(63, 189, 92);</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>661</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../v.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>50</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 238, 240);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../ACCOUNT.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_8</zorder>
     <zorder>lineEdit_3</zorder>
     <zorder>lineEdit_4</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>label_9</zorder>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Les spectacles</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mudir MT</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">alternate-background-color: rgb(255, 255, 255);
background-color: rgb(237, 238, 240);</string>
      </property>
      <property name="text">
       <string>Liste des spectacles :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qb_ajouter">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>230</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supp">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QFontComboBox" name="fontComboBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 182, 182);
background-color: rgb(225, 225, 225);</string>
      </property>
      <property name="currentText">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>250</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mudir MT</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(231, 232, 236);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>260</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_horaire">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_duree">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>170</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>140</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 230, 234);</string>
      </property>
      <property name="text">
       <string>horaire</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>170</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(223, 227, 230);</string>
      </property>
      <property name="text">
       <string>duree</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_Num">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>110</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 234, 238);</string>
      </property>
      <property name="text">
       <string>Num</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-7</y>
        <width>661</width>
        <height>331</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../bb.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Exporter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>200</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_type">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_spec">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>291</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="num_supp">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <zorder>label_10</zorder>
     <zorder>label</zorder>
     <zorder>qb_ajouter</zorder>
     <zorder>pb_supp</zorder>
     <zorder>fontComboBox</zorder>
     <zorder>label_2</zorder>
     <zorder>spinBox</zorder>
     <zorder>le_horaire</zorder>
     <zorder>le_duree</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>le_Num</zorder>
     <zorder>label_7</zorder>
     <zorder>pushButton</zorder>
     <zorder>label_4</zorder>
     <zorder>le_type</zorder>
     <zorder>tab_spec</zorder>
     <zorder>num_supp</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Notification</string>
     </attribute>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>521</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mudir MT</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(236, 237, 239);</string>
      </property>
      <property name="text">
       <string>Historique des notifications :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QScrollBar" name="horizontalScrollBar_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>481</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(199, 199, 199);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBar_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>70</y>
        <width>16</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(206, 206, 206);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-7</y>
        <width>661</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../bb.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <zorder>label_11</zorder>
     <zorder>listView</zorder>
     <zorder>label_3</zorder>
     <zorder>dateEdit</zorder>
     <zorder>horizontalScrollBar_2</zorder>
     <zorder>verticalScrollBar_2</zorder>
     <zorder>pushButton_4</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
